create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusConstants.java

```diff
@@ -7,6 +7,9 @@
     public static final String ADDITIONAL_IDENTIFIERS_SCOPUS_ID_SOURCE_NAME = "scopusIdentifier";
     public static final String SCOPUS_ITEM_IDENTIFIER_SCP_FIELD_NAME = "scp";
 
+    //URI constants:
+    public static final String DOI_OPEN_URL_FORMAT = "https://doi.org";
```

Comment:
Could not find this constant anywhere else in the project so created my own

Reply:
Add the trailing slash

Reply:
If I understood it correctly I can avoid the trailing slash / or magic string slash altogheter by using the UriWrapper.

(coming from the frontend, devs, I don't like to add ending slashes to URL's, it's so quick to end up with generating urls with double slashes. ex. https://doi.org//something. I rather work around that)